list_of_prime_numbers/main.py

```diff
@@ -1,10 +1,61 @@
+"""
+read code on this link to know various functions which are faster -
+https://stackoverflow.com/questions/2068372/fastest-way-to-list-all-primes-below-n
+and
+https://www.daniweb.com/programming/software-development/code/462120/isprime-function-python - info on how to use a
+output from 1 function into another.
 
 def _is_prime(number):
-    pass
+    if number == 2 or number == 3: return True
+    if number < 2 or number % 2 == 0: return False
+    if number < 9: return True
+    if number % 3 == 0: return False
+    r = int(number ** 0.5)
+    f = 5
+    while f <= r:
+        print
+        '\t', f
+        if number % f == 0: return False
+        if number % (f + 2) == 0: return False
+        f += 6
+    return True
+"""
+
+
+def _is_prime(number):
+    if number < 2:
+        return False
+    elif number == 2:
+        return True
+    for n in range(2, number):
+        if number % n ==0:
+            return False
+    return True
+
+
+number = 19
+testcode = _is_prime(number)
+print(testcode)
 
 
 def list_of_prime_numbers(max_number):
-    pass
+    primes_list = []
+    new_max = max_number + 1
+    for n in range(2, new_max):
+        if _is_prime(n):
+            primes_list.append(n)
+    return primes_list
+
```

Comment:
prime_numbers = []

    for num in range(max_number + 1):
        if _is_prime(num):
            prime_numbers.append(num)

    return prime_numbers 

We ended up with some very similar solutions here. The code snippet above is what I came up with.

Reply:
Thanks! for reviewing and solution compares. It helps me understand different approach to getting to a same result.